service/collector/collector.go

```diff
@@ -19,55 +19,75 @@ const (
 	Namespace          = "aws_operator"
 )
 
+const (
+	ClusterTag      = "giantswarm.io/cluster"
+	InstallationTag = "giantswarm.io/installation"
+	OrganizationTag = "giantswarm.io/organization"
```

Comment:
Where is this used?
Are those new style tags? Is @giantswarm/sig-monitoring aware of that?

Reply:
The tags only got moved from one file to another.

Reply:
Ack. I'm not aware how they are used.